tensorflow/lite/kernels/round_test.cc

```diff
@@ -25,11 +27,12 @@ namespace {
 
 using ::testing::ElementsAreArray;
 
+template <typename T>
 class RoundOpModel : public SingleOpModel {
  public:
   RoundOpModel(std::initializer_list<int> input_shape, TensorType input_type) {
-    input_ = AddInput(TensorType_FLOAT32);
-    output_ = AddOutput(TensorType_FLOAT32);
+    input_ = AddInput(input_type);
```

Comment:
Since we're templating the class based on type now, can we just use `typeToTfLiteType()` here and omit the argument? Avoids getting out of sync.